tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -46,6 +46,67 @@ use fmt::{Debug, Display};
 /// This trait is already implemented for function pointers with the same
 /// signature as `format_event`.
 ///
+/// # Example implementation
```

Comment:
Nit: for consistency we tend to just prefer 
```suggestion
/// # Examples
```